source/CardGames.py

```diff
@@ -205,20 +205,32 @@ def start_game(self):
 
         while self.done != True:
             #Different rule sequence for first turn
-            while first_turn:
+            #while first_turn:
                 #Fugitive places 1 or 2 new hideouts
-                fugitive_choices = self.fugitive_first_turn()
+                #fugitive_choices = self.fugitive_first_turn()
                 #Marshall draws 2 cards; chooses which deck to draw from
-                print(f"{self.marshall.name} is next!")
-                marshall_choices = self.marshall_first_turn()
+                #print(f"{self.marshall.name} is next!")
+                #marshall_choices = self.marshall_first_turn()
                 
             #Outside of first turn
             #Fugitive draws 1 card from any deck. 
             #Can choose to place 1 new hideout, or pass
-            
+            #self.fugitive_repeating_turn()
+              
             #Marshall draws 1 card from any deck.
             #Makes a singular guess to find a hideout (possible expand into guessing multiple hideouts)
-
+            print("<<<Marshall's Turn>>>")
+            response = ""
+            while response != "y" and "n":
```

Comment:
Good idea to make this an option brother! Good work!

Reply:
Thank you brother!